extensions/replicate/package.json

```diff
@@ -16,10 +17,10 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "replicate",
-      "title": "View Predictions",
-      "subtitle": "Replicate",
-      "description": "View your Replicate predictions",
+      "name": "index",
```

Comment:
Could we NOT change this if it's not really important? When this is changed people loses their hotkey/aliases/ranking for the command

Reply:
@cbh123 Do you think this new feature should be a separate command? So users would have one command for viewing past predictions and another to create new ones?

Reply:
Sure! Just pushed a commit to keep the command consistent. I think having one command is good for now so we can get this merged in!